test/CompanyService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs

```diff
@@ -22,51 +23,39 @@ namespace LT.DigitalOffice.CompanyService.Business.UnitTests.Commands.Department
     internal class CreateDepartmentCommandTests
     {
         private ICreateDepartmentCommand _command;
-        private Mock<IDepartmentRepository> _repositoryMock;
-        private Mock<IDepartmentUserRepository> _userRepositoryMock;
-        private Mock<ICompanyRepository> _companyRepositoryMock;
-        private Mock<IAccessValidator> _accessValidatorMock;
-        private Mock<ICreateDepartmentRequestValidator> _validatorMock;
-        private Mock<IDbDepartmentMapper> _mapperMock;
+        private AutoMocker _autoMock;
 
         private CreateDepartmentRequest _request;
         private DbDepartment _dbDepartment;
 
+        private readonly Guid _directorId = new Guid("528E413B-7A3D-4CA9-851A-B9DA5840B216");
```

Comment:
Why you write guid like this? You can write Guid.NewGuid

Reply:
Done